Gateway/Request/APMBuilder.php

```diff
@@ -280,7 +280,7 @@ public function build(array $buildSubject)
                 break;
             case Shopeepay::CODE:
                 $paymentInfo[self::SOURCE] = [
-                    self::SOURCE_TYPE => 'shopeepay',
+                    self::SOURCE_TYPE => $this->helper->isMobilePlatform() ? 'shopeepay_jumpapp' : 'shopeepay',
```

Comment:
can you make sure `shopeepay_jumpapp` is exist first bc 
not every one who have shopeepay (direct version) is will have jump app enable